### DISCUSSION TITLE
Issues with VIM and MC in urxvt256 terminal

### DISCUSSION THREAD

**Discussion by blangguth** [dracula/dracula-theme#812] [Q&A] [answered]
Hello!
At first.. I love the Dracula Theme and try to install it consequently across my system.

But I have some issues with my Terminal running Apps.

I appended my .Xressources accordingly.

`44 ! Cursor
 45 urxvt.cursorBlink: true
 46 urxvt.cursorColor: #50FA7B
 47 urxvt.cursorUnderline: true
 48 
 49 ! Dracula Xresources palette
 50 *.foreground: #F8F8F2
 51 *.background: #282A36
 52 *.color0:     #000000
 53 *.color8:     #4D4D4D
 54 *.color1:     #FF5555
 55 *.color9:     #FF6E67
 56 *.color2:     #50FA7B
 57 *.color10:    #5AF78E
 58 *.color3:     #F1FA8C
 59 *.color11:    #F4F99D
 60 *.color4:     #BD93F9
 61 *.color12:    #CAA9FA
 62 *.color5:     #FF79C6
 63 *.color13:    #FF92D0
 64 *.color6:     #8BE9FD
 65 *.color14:    #9AEDFE
 66 *.color7:     #BFBFBF
 67 *.color15:    #E6E6E6
 68 
 69 urxvt*underlineColor: #BD93F9
`

then I installed the VIM and MC themes.
But I experience issues with some colors accidentally behaving as background.
![2023-01-25-222941_1073x750](https://user-images.githubusercontent.com/718902/214695690-ae50e5ce-95b3-4008-8e71-058ccf8aa0f5.png)
![2023-01-25-223506_675x744](https://user-images.githubusercontent.com/718902/214696584-bba0b04f-7ba0-4e72-adf1-485df639e1a2.png)

I desperately tried to exchange colors - without much luck.

Any ideas?

Thanks!
~Ben

**Comment by benknoble** [comment, marked as answer]
For some reason I can't transfer this to `dracula/vim` where it belongs, but see https://github.com/dracula/vim/issues/265.

**Reply by blangguth** [reply]
That worked great! Thanks a lot!
